### PR TITLE
[FEAT]lesson-10-task-1

### PR DESCRIPTION
**Pull request by heilrake**
lesson 10 task 1
image

### CODE REVIEW

File: src/lesson/Lesson10/Task1/Index.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import TableOfUsers from './component/TableOfUsers';
```
Comment (daviddubliakov) [inline src/lesson/Lesson10/Task1/Index.jsx:3]: enter between component and lib imports

**Review by daviddubliakov — CHANGES_REQUESTED** [review]
Please rename PR to "[FEAT] lesson 10 task 1"
Leaved one comment
FIX CONFLICTS

File: src/lesson/Lesson10/Task1/Index.jsx (hunk 1 above)
Comment (AI-Avanger) [reply src/lesson/Lesson10/Task1/Index.jsx:3]: fixed

**Review by daviddubliakov — CHANGES_REQUESTED** [review]
Fix row with the links

**Pull request merged**